suite/secure-storage-webauthn/src/webauthnSecureStorage.ts

```diff
@@ -0,0 +1,17 @@
+import {
+    EncryptableBranded,
+    EncryptedHex,
+    SecureStorage,
+    asEncryptedHex,
+} from '@suite-common/secure-storage';
+import { ok } from '@trezor/type-utils';
+
+export const createWebauthnSecureStorage = (): SecureStorage => ({
+    encrypt: async <T extends EncryptableBranded>({ value }: { value: T }) =>
+        // Todo: implement
+        ok(asEncryptedHex(await Promise.resolve(value as T))),
+
+    decrypt: async <T extends EncryptableBranded>({ value }: { value: EncryptedHex<T> }) =>
+        // Todo: implement
+        ok(await Promise.resolve(value as unknown as T)),
```

Comment:
this inlined `await Promise.resolve` is reaaally weird and probably unnecessary.
I know this is temporary code, so it's a nit – but also, all the more reason to KISS if it's temporary.

```suggestion
    encrypt: <T extends EncryptableBranded>({ value }: { value: T }) =>
        // Todo: implement
        Promise.resolve(ok(asEncryptedHex(value as T))),

    decrypt: <T extends EncryptableBranded>({ value }: { value: EncryptedHex<T> }) =>
        // Todo: implement
        Promise.resolve(ok(value as unknown as T)),
```